lm_eval/tasks/complete_smile.py

```diff
@@ -108,10 +115,5 @@ def higher_is_better(self):
         }
 
     def _valid_mol_eval(self, smile):
-        try:
-            m1 = Chem.MolFromSmiles(smile)
-        except:
-            return 0
-        if m1 is None:
-            return 0
-        return 1
+        is_valid = Chem.MolFromSmiles(smile)
```

Comment:
Any idea why the `except` was there in the first place and hence why it's ok to drop it now?

Reply:
I think we suppressed the errors when we added `RDLogger.DisableLog('rdApp.*') ` at the start so now we can just use the behaviour which is specified in the [documentation for this library](https://www.rdkit.org/docs/source/rdkit.Chem.rdmolfiles.html) which says that MoleFromSmiles `RETURNS a Mol object, None on failure.`